cmf-cli/resources/template_feed/grafana/Grafana.Package/cmfpackage.json

```diff
@@ -0,0 +1,29 @@
+{
+  "packageId": "<%= $CLI_PARAM_CustomPackageName %>",
+  "version": "<%= $CLI_PARAM_CustomPackageVersion %>",
+  "description": "Cmf Custom <%= $CLI_PARAM_Tenant %> <%= $CLI_PARAM_CustomPackageName %> Package",
+  "packageType": "Grafana",
+  "isInstallable": true,
+  "isUniqueInstall": true,
+  "steps": [
+    {
+      "order": "1",
+      "type": "DeployFiles",
+      "ContentPath": "**/**"
+    },
+    {
```

Comment:
Since we are not scaffolding the clickhouse plugin by default we can assume the datasource.yaml is not a tagged file.

When clickhouse plugin was set in datasource.yaml, we used to need to change de Default Database tag. Now, we don't need that.

Reply:
It would be a DeployFile then?